modules/images/image-loading-optimization/storage/rest-api.php

```diff
@@ -0,0 +1,158 @@
+<?php
+/**
+ * REST API integration for the module.
+ *
+ * @package performance-lab
+ * @since n.e.x.t
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+const ILO_REST_API_NAMESPACE = 'image-loading-optimization/v1';
+
+const ILO_PAGE_METRIC_STORAGE_ROUTE = '/image-loading-optimization/page-metric-storage';
```

Comment:
Two points of feedback on the route name:
* The `image-loading-optimization` is already in the namespace, so I think we should drop it from the route name.
* To better follow [resource-oriented design](https://google.aip.dev/121), I would advise we center the naming around the resource, which is the "page metrics". I realize we don't need to provide API endpoints for CRUD operations on the individual custom post type entries for now, but we could choose a name here that would allow ourselves to do that in the future: We could follow [this custom methods recommendation](https://google.aip.dev/136) and go with:
```suggestion
const ILO_PAGE_METRIC_STORAGE_ROUTE = '/page-metrics:store';
```

Reply:
Oops, missed the redundancy. I think it's because I originally had the namespace for PerfLab.

As for the route, how about just `/page-metrics`? (Or `/url-metrics`?) I don't think we need to incorporate any custom method here as the resource is `/page-metrics` and the method of `POST` is sufficiently semantic. And WP REST API routes don't typically include colons like that, as far as I'm aware.

Reply:
Fixed in 7d5b2f5

Reply:
Left a follow up comment in https://github.com/WordPress/performance/pull/878#discussion_r1394635453.